.circleci/config.yml

```diff
@@ -23,14 +23,24 @@ jobs:
     - run: git diff --exit-code
 
   build:
-    executor: golang
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
can't we use 1804?

Reply:
it's not listed in the [docs](https://circleci.com/docs/2.0/configuration-reference/#available-machine-images).